<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/resources/resources/back.png</normaloff>:/resources/resources/back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="windowLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>881</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>270</y>
     <width>501</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>10</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/resources/resources/start.png</normaloff>:/resources/resources/start.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPathA">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelPathA">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Path A</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPathB">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Path B</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPathB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="browserPathAButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>250</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/resources/resources/folder.png</normaloff>:/resources/resources/folder.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="browserPathBButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>310</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/resources/resources/folder.png</normaloff>:/resources/resources/folder.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="browserPathResultButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>370</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/resources/resources/folder.png</normaloff>:/resources/resources/folder.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="labelPathResult">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Path result</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPathResult">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelBackButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelStartButton">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>70</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxOutput">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>240</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Show output</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditElementSize">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>610</y>
     <width>181</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>16</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelElementSize">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Element size</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelReadBufferSize">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>590</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Read buffer size</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditReadBufferSize">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>610</y>
     <width>181</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>100000</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditWriteBufferSize">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>680</y>
     <width>181</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>50000</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelWriteBufferSize">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>660</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Write buffer size</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelThreads">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Threads</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditThreads">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNumberOfBuckets">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>151</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Divisor of elem. #</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNumberOfBuckets">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of buckets</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditQueueBufferSize">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>680</y>
     <width>181</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>100000</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelQueueBufferSize">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>660</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Queue buffer size</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPathBuckets">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelPathBuckets">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Path buckets</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browserPathBucketsButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>490</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/resources/resources/folder.png</normaloff>:/resources/resources/folder.png</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditHashSeed1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>311</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>77777777777777777777777777777777</string>
   </property>
   <property name="maxLength">
    <number>32</number>
   </property>
   <property name="placeholderText">
    <string>16-Byte</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSeed1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Hash seed #1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditHashSeed2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>190</y>
     <width>311</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>0123456789ABCDEF0123456789ABCDEF</string>
   </property>
   <property name="maxLength">
    <number>32</number>
   </property>
   <property name="placeholderText">
    <string>16-Byte</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSeed2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>170</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Hash seed #2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditHashSeed3">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>190</y>
     <width>311</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC</string>
   </property>
   <property name="maxLength">
    <number>32</number>
   </property>
   <property name="placeholderText">
    <string>16-Byte</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSeed3">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>170</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Hash seed #3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTableSizeMultiplier">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>1.2</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTableSizeMultiplier">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Table size multiplier</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditRecursiveDeepnessLimit">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>120</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>2000</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRecursiveDeepnessLimit">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>100</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Deepness limit</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditIP">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>120</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelIP">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>100</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>IP</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPathResultB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelPathResultB">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Path result B</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browserPathResultButtonB">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>430</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/resources/resources/folder.png</normaloff>:/resources/resources/folder.png</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPort">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>120</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelPort">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>100</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTableSize">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>120</y>
     <width>181</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Stronger then multiplier</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTableSize">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>100</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Table size</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
